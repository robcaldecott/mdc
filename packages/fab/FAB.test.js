import React from "react";
import { FAB } from ".";
import renderer from "react-test-renderer";

it("renders", () => {
  expect(renderer.create(<FAB>favorite</FAB>)).toMatchSnapshot();
});

it("renders mini", () => {
  expect(renderer.create(<FAB mini>favorite</FAB>)).toMatchSnapshot();
});

it("renders exited", () => {
  expect(renderer.create(<FAB exited>favorite</FAB>)).toMatchSnapshot();
});
