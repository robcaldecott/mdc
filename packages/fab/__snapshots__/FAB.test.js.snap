// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<button
  aria-label="favorite"
  className="mdc-fab material-icons"
>
  <span
    className="mdc-fab__icon"
  >
    favorite
  </span>
</button>
`;

exports[`renders exited 1`] = `
<button
  aria-label="favorite"
  className="mdc-fab material-icons mdc-fab--exited"
>
  <span
    className="mdc-fab__icon"
  >
    favorite
  </span>
</button>
`;

exports[`renders mini 1`] = `
<button
  aria-label="favorite"
  className="mdc-fab material-icons mdc-fab--mini"
>
  <span
    className="mdc-fab__icon"
  >
    favorite
  </span>
</button>
`;
